cli/dcoscli/data/config-schema/cosmos.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "url": {
+      "type": "string",
+      "format": "uri",
+      "title": "Cosmos base URL",
+      "description": "Base URL for talking to COSMOS. It overwrites the value specified in core.dcos_url",
+      "default": "http://localhost:7070"
+    }
+  },
+  "additionalProperties": false
+}
```

Comment:
ok. I'm going to remove `package.sources` and `package.cache` in a following commit because it makes this a much larger change.